index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Form</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <div class="top">
+    <h1>freeCodeCamp Survey Form</h1>
+    <p><em>Thank you for taking the time to help us improve the platform</em></p>
+  </div>
+
+  <div class="main-form">
+    <form class="" action="/" method="post">
```

Comment:
To comply with html and css best practices, you can use the main tag to enclose the main content of the page

Reply:
- [ ] Good one here also. Nice feedback but it could be very explicit to add why they should use the `main` tag.
- What if I've forgotten about the `HTML and CSS best practice`?  I definitely will need a reminder. 
KIndly, add links to the `Html and CSS best practice` so that the author of the code can have a better understanding of your proposed changes.

Reply:
- [ ] Good one here also. Nice feedback but it could be very explicit to add why they should use the `main` tag.
- What if I've forgotten about the `HTML and CSS best practice`?  I definitely will need a reminder. 
KIndly, add links to the `Html and CSS best practice` so that the author of the code can have a better understanding of your proposed changes.

Reply:
In this webpage, the form is what holds the main content of the page. In other words, its the "meat" of the webpage. In order to optimize your webpage, you can enclose that section with the main tag. You can read more about how the[ main ](https://www.w3docs.com/learn-html/html-main-tag.html)tag works here and the HTML and CSS best practices [here ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)and see ways in which to improve that.